index.json

```diff
@@ -699,18 +689,57 @@
         "very noisy",
         "hot",
         "basic",
-        "default",
         "betaflight"
       ],
       "author": "Betaflight",
       "priority": 100
     },
+    {
+      "fullPath": "presets/4.3/filters/ctzsnooze_race.txt",
+      "hash": "3cd6b38b21527d3acb44b6e73e0df0816f9bef36ad1c4a83644d51a903aa579b",
+      "title": "ctzsnooze's race filter settings",
```

Comment:
Maybe we should use Title Case in all titles.

Just trying to make the content more consistent.